contrib/pyln-client/requirements.txt

```diff
@@ -1 +1,2 @@
 recommonmark>=0.7.*
+pyln-bolt7==1.0.1.*
```

Comment:
better avoid fixing the version to a minimum requirement version?

Maybe it is better use the `pyln-bolt7>=1.0.1.7` to avoid that pip said "hey there is a dependence that wants a new package".

Reply:
Just thinking too late, maybe the dependences are skipped (not sure what dependences), because there is this type of conflict?

```
The conflict is caused by:
    The user requested pyln-proto 0.10.1 (from /home/runner/work/lightning/lightning/contrib/pyln-proto)
    pyln-bolt7 1.0.1.137 depends on pyln.proto==0.8.4
```

https://github.com/ElementsProject/lightning/pull/4582/checks?check_run_id=3512149678#step:5:222

Reply:
Here we go, it is my fault, we upgrade the pyln-proto >= 0.9.3 https://github.com/ElementsProject/lightning/pull/4679, this pull from pip the last version, so pyln-proto 0.10.1. However, on the pip repository, we miss the last version of the module, as @cdecker points out in the last meeting last week.

They are also a couple of messages on the [IRC log](https://gnusha.org/c-lightning/2021-08-23.log) where is mentioned this problem of dependencies missing.


```
04:15 < vincenzopalazzo> Guys stupid question, the new version of the package pyln-sec it is update with the new c-lightning version? because I'm getting the following warning `ERROR: pyln-bolt1 1.0.1.137 has requirement pyln.proto==0.8.4, but you'll have pyln-proto 0.10.1 which is incompatible.` but it think this is over with the following PR https://github.com/ElementsProject/lightning/pull/4679/files
04:19 < cdecker[m]> That's because the version on pypi is really old, and still predates that PR. I'll look into automating deployments of the pypi artifacts via GH actions
04:31 -!- rusty [~rusty@103.93.169.121] has quit [Quit: Leaving.]
04:33 < vincenzopalazzo> Christian Decker: Thanks for this :) I'm try to fix the warning in lnprototest
```

Sorry about that, a solution that I found to make CI happy again, at least of python dependencies is to remove the dependencies in this `requirements.txt` file, and all look fine again, but only with CI (see https://github.com/vincenzopalazzo/lightning/runs/3512820849#step:5:2194).